src/Svelte/assets/package.json

```diff
@@ -5,6 +5,7 @@
     "module": "dist/register_controller.js",
     "version": "1.0.0",
     "license": "MIT",
+    "type": "module",
```

Comment:
@stof I'm wondering if I could get your advice. This was needed to get `@sveltejs/vite-plugin-svelte` to play nicely. In reality, we are already using `import` inside all packages and our users are using `import`. So, do you have any concern about this change? If not, we should apply it equally across all packages.